src/next-env.d.ts

```diff
@@ -0,0 +1,6 @@
+/// <reference types="next" />
+/// <reference types="next/types/global" />
+/// <reference types="next/image-types/global" />
+
+// NOTE: This file should not be edited
+// see https://nextjs.org/docs/basic-features/typescript for more information.
```

Comment:
What is it for? tests seems to be green even without it. This also causes the type error in `src/setEnvVars.ts`.

Reply:
Typescript fails without this.

Reply:
I see now. It only fails when `tsc --noEmit`. No issues at tests or build time. So this is a new file Next.js users are supposed to keep in their userland projects, right? Using `skipLibCheck` means that all `next-page-tester` would be also forced to enable the option?

In the worst case we can keep it and `// @ts-expect-error` the error. How do you see it?